### DISCUSSION TITLE
EISeg high resolution images

### DISCUSSION THREAD

**Discussion by lumaIPU** [PaddlePaddle/PaddleSeg#2750] [Ideas]
Hi all,
I've tried EISeg and I think it's really useful and well done.

However, I've noticed that automatic segmentation with high resolution images is much more accurate when the image gets divided in a grid. Depending on the image size and content, it could often be hard to understand the content of each grid cell, so I think it could be useful to have the option to change the grid size (number of cells). Is there a way to do it?

It would be even better to have the same accuracy that you get in grid mode but with the full resolution image. A solution could be to crop an area around the selected point and feed only that to the segmentation model?

I've also tried to scale down the image before opening it with EISeg but the model struggles to detect small objects and the overall accuracy is not satisfying. 

Thank you for the great tool!

**Comment by haoyuying** [comment]
Next version we will consider this feature, thank you for your advise.
